Sources/Rendering/OpenGL/ImageMapper/index.js

```diff
@@ -920,9 +920,7 @@ function vtkOpenGLImageMapper(publicAPI, model) {
     }
 
     // rebuild the VBO if the data has changed
-    const toString = `${slice}A${image.getMTime()}A${imgScalars.getMTime()}B${publicAPI.getMTime()}C${model.renderable.getSlicingMode()}D${actor
-      .getProperty()
-      .getMTime()}`;
+    const toString = `${slice}A${image.getMTime()}A${imgScalars.getMTime()}B${publicAPI.getMTime()}C${model.renderable.getSlicingMode()}`;
```

Comment:
This is testing if the property has been updated and for good measure. The texture filtering changes based on property interpolation. The property should not be changing each frame. If you see that the mapper is re-creating the texture each frame because of the property, check to see if something else in your application (or vtk-js) is changing the property.

Reply:
it is not my application, it can be replicated in the ImageMapper demo as I showed in the issue. basically window levelling should not re create scalar texture as it doesn't for VolumeMapper either. So maybe windowLevelling should not update the actor properties?

Reply:
Yes, or better yet - this `if` check should just test whether the property's interpolation has changed.

Reply:
@sankhesh How do you suggest I filter the changed properties? is there any way you recommend?